### PR TITLE
chore: Migrate tutorial to the Verax GitHub organization

### PR DESCRIPTION
**Pull request by alainncls**
## What does this PR do?

- Removes the `tutorial`package from the monorepo
- Migrates the tutorial to [its own repo](https://github.com/Verax-Attestation-Registry/verax-tutorial) in the Verax GitHub organization

### Type of change

- [X] Chore
- [ ] Bug fix
- [ ] New feature
- [ ] Documentation update

## Check list

- [ ] Unit tests for any smart contract change
- [ ] Contracts and functions are documented

### CODE REVIEW

**Comment by orbmis** [comment]
What's the eli5 of this PR @alainncls ?

**Review by satyajeetkolhapure — APPROVED** [review]
